### PR TITLE
Add baffle.js@0.3.6 w/ npm autoupdate

### LINKED ISSUE #9254: [Request] Add angular-mapboxgl-directive
**Library name:** angular-mapboxgl-directive
**Git repository url:** https://github.com/Naimikan/angular-mapboxgl-directive
**npm package url(optional):** https://www.npmjs.com/package/angular-mapboxgl-directive
**License(s):** MIT
**Official homepage:** https://naimikan.github.io/angular-mapboxgl-directive/#/
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by extend1994**
PR for #9529

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/camwiegert/baffle
 * Official website (optional, not the repository): https://camwiegert.github.io/baffle/
 * NPM package url (optional): https://www.npmjs.com/package/baffle
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 999
   - Count of watchers: 22
   - Count of forks: 44
 * NPM download stats (optional):
   - Downloads in the last day: 17
   - Downloads in the last week: 139
   - Downloads in the last month: 448

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9254, cc @Naimikan

### CODE REVIEW

File: ajax/libs/baffle.js/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  },
+  "keywords": [
+    "baffle",
+    "tiny",
+    "simple",
+    "javascript library",
+    "obfuscate and reveal text"
+  ],
+  "homepage": "https://camwiegert.github.io/baffle",
+  "babel": {
```
Comment (PeterDaveHello) [inline ajax/libs/baffle.js/package.json]: Please remove "babel"

Comment (extend1994) [inline ajax/libs/baffle.js/package.json]: I have removed it.

**Comment by extend1994** [comment]
@PeterDaveHello Please help me review this pull request, thank you!

File: ajax/libs/baffle.js/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/camwiegert/baffle.git"
+  },
+  "keywords": [
+    "baffle",
+    "tiny",
+    "simple",
+    "javascript library",
+    "obfuscate and reveal text"
```
Comment (PeterDaveHello) [inline ajax/libs/baffle.js/package.json]: Try to separate to `obfuscate`, `reveal text`

File: ajax/libs/baffle.js/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "description": "A tiny javascript library for obfuscating and revealing text in DOM elements.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/camwiegert/baffle.git"
+  },
+  "keywords": [
+    "baffle",
+    "tiny",
+    "simple",
+    "javascript library",
```
Comment (PeterDaveHello) [inline ajax/libs/baffle.js/package.json]: This is not like keyword.

File: ajax/libs/baffle.js/package.json (hunk 2 above)
Comment (PeterDaveHello) [inline ajax/libs/baffle.js/package.json]: Maybe add one more `"DOM"`

File: ajax/libs/baffle.js/package.json (hunk 3 above)
Comment (extend1994) [inline ajax/libs/baffle.js/package.json]: Do you mean all of line 11 to line 14?

Comment (PeterDaveHello) [inline ajax/libs/baffle.js/package.json]: Line 14 - "javascript library"

**Comment by extend1994** [comment]
@PeterDaveHello please help me review this pull request again, thank you.

**Comment by PeterDaveHello** [comment]
Actually I think `reveal text` should be `reveal`, that's my fault :sweat_smile:

**Pull request merged**